src/AdaProx.jl

```diff
@@ -272,6 +272,43 @@ function stepsize(rule::OurRule, (gamma1, gamma0), x1, grad_x1, x0, grad_x0)
     return (gamma, sigma), (gamma, gamma1)
 end
 
+
+
+struct OurRulePlus{R}
+    gamma::R
+    xi::R
+    nu::R
+    r::R
+end
+
+function OurRulePlus(; gamma = 0, nu = 1, xi = 1, r = 1/2)
+    _gamma = if gamma > 0
+        gamma
+    else
+        error("you must provide gamma > 0")
+    end
+    R = typeof(_gamma)
+    return OurRulePlus{R}(_gamma, R(xi), R(nu), R(r))
+end
+
+function stepsize(rule::OurRulePlus)
+    gamma = rule.gamma
+    return (gamma, gamma), (gamma, gamma)
```

Comment:
Then again this is because in the near future we should add the primal-dual stepsize version with first (gamma, gamma) transforming into (gamma, sigma).